Fixed #28344 -- Allowed customizing queryset in Model.refresh_from_db()/arefresh_from_db(). (#17592)
django/django: docs/releases/5.1.txt

  :class:`~django.db.models.expressions.ValueRange` allows excluding rows,
  groups, and ties from the window frames.

> * The new ``from_queryset`` argument of :meth:`.Model.refresh_from_db` can be used

Reviewer: ```suggestion
* The new ``from_queryset`` argument of :meth:`.Model.refresh_from_db` allows
   customizing ...
```

  to reload model using non-default Manager, to lock the row before reloading, and

>   to select additional related objects (:ticket:`28344`).

Reviewer: We don't reference tickets for new features.
```suggestion
  to select additional related objects.
```
